packages/hardhat-core/src/internal/hardhat-network/provider/node-types.ts

```diff
@@ -17,17 +17,17 @@ export function isForkedNodeConfig(
 
 interface CommonConfig {
   automine: boolean;
-  blockGasLimit: bigint;
+  blockGasLimit: number;
```

Comment:
These can be `bigint`, as it's an internal type. If it's more convenient to revert to `number` that's fine.